helm/openwhisk/templates/elasticsearch-svc.yaml

```diff
@@ -30,6 +30,15 @@ metadata:
   annotations:
     service.alpha.kubernetes.io/tolerate-unready-endpoints: "true"
 spec:
+  type: {{ .Values.elasticsearch.service.type }}
```

Comment:
Should this be the `ClusterIP` if no type is specified?

Reply:
Yes, ClusterIP is default service type.